.trybe/requirements.json

```
@@ -2,82 +2,82 @@
   "requirements": [
     {
       "identifier": "test_validar_conteudo_do_arquivo_gerado",
-      "description": "Executar o método `analyse_log` deverá preencher corretamente os dados no arquivo `data/mkt_campaign.txt`",
+      "description": "1.1 - Será validado se ao executar o método `analyze_log` os dados são preenchidos de forma correta no arquivo `data/mkt_campaign.txt`",
       "bonus": false
     },    
     {
       "identifier": "test_validar_arquivo_inexistente",
-      "description": "Executar o método `analyse_log` com um arquivo inexistente deverá retornar um erro",
+      "description": "1.2 - Será validado se ao executar o método `analyze_log` com um arquivo inexistente deverá retornar um erro",
       "bonus": false
     },
     {
       "identifier": "test_validar_arquivo_com_extensao_invalida",
-      "description": "Executar o método `analyse_log` com uma extensão inválida deverá retornar um erro",
+      "description": "1.3 - Será validado se ao executar o método `analyze_log` com uma extensão inválida deverá retornar um erro",
```

```suggestion
      "description": "1.3 - Será validado se, ao executar o método `analyze_log` com uma extensão inválida, o método retorna um erro",
```

Em todos os textos é importante o `ao executar o método lalala` vir companhado de um `o método retorna um...` ao invés de `deverá retornar`, `deve adicionar`... O sujeito da frase no final é sempre a função! Tem que ajustar em alguns lugares isso!